test/dkg_handler.erl

```diff
@@ -21,7 +21,8 @@
           privkey :: undefined | tpke_privkey:privkey() | tpke_privkey:privkey_serialized(),
           n :: pos_integer(),
           t :: pos_integer(),
-          id :: pos_integer()
+          id :: pos_integer(),
+          bomb = 10 :: integer()
```

Comment:
What is the bomb?

Reply:
Still want to know what the bomb is

Reply:
half-baked idea that never went anywhere.  removing.